engine/orchestration/src/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1090,8 +1091,8 @@ public void orchestrateStart(final String vmUuid, final Map<VirtualMachineProfil
                         break;
                     }
 
-                } catch (final OperationTimedoutException e) {
-                    s_logger.debug("Unable to send the start command to host " + dest.getHost());
+                } catch (OperationTimedoutException e) {
+                    s_logger.debug("Unable to send the start command to host " + dest.getHost()+" failed to start VM: "+vm.getUuid());
```

Comment:
Hi wido,
most of these kind of logs in cloudstack are at debug level, as in it lets us debug why a vm deployment has failed. In this case the error will be vm deployment failed.